docs/guides/migrate-from-spectral.md

```diff
@@ -0,0 +1,131 @@
+# Migrate to Redocly from Spectral
+
+[Spectral](https://stoplight.io/open-source/spectral) offers similar linting capabilities to Redocly CLI and the rest of the Redocly tools. This guide lays out the differences so you can switch tools if you want to.
```

Comment:
```suggestion
[Spectral](https://stoplight.io/open-source/spectral) offers similar linting capabilities to Redocly CLI. This guide lays out the differences so you can switch tools if you want to.
```

I wonder if that "and the rest of the Redocly tools" adds anything? Also, I'm not sure what other capability it would be similar to besides the CLI lint command?

Reply:
The linting capabilities in Redocly CLI, Workflows, and whatever else are similar to what Spectral offers. I was trying to make the point that it is not just the CLI, but all the Redocly tools, if people adopt all of them.